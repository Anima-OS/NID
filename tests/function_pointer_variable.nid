// function pointer variable
int (*some_callback)(int a, int b);

