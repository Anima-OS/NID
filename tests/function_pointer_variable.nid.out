{ declarations: 
   [ { variable: 
        { type: { name: 'int' },
          attributes: {},
          name: 'some_callback',
          parameters: 
           [ { type: { name: 'int' }, name: 'a', attributes: {} },
             { type: { name: 'int' }, name: 'b', attributes: {} } ],
          functionPointer: true } } ] }
